File: Python/abc030A.py
n = input().split()
t = int(n[1]) / int(n[0])
a = int(n[3]) / int(n[2])
if t < a:
  print("AOKI")
if t > a:
  print("TAKAHASHI")
if t == a:
  print("DRAW")

File: JavaScript/abc030A.js
function Main(input) {
  const a = input.split(' ');
  if(a[1] / a[0] < a[3] / a[2]){
    console.log("AOKI")
  }else if(a[1] / a[0] > a[3] / a[2]){
    console.log("TAKAHASHI")
  }else{
    console.log("DRAW")
  }
}

Main(require("fs").readFileSync("/dev/stdin", "utf8"));
